### DISCUSSION TITLE
游戏好少啊！steam版的异域：镇魂曲加强版有drm吗？

### DISCUSSION THREAD

**Discussion by ghost** [Gamuxorg/bbs#356] [投递和请求游戏]
没有drm我可以打包一个 中文的

**Comment by shouhuanxiaoji** [comment]
不是很清楚。可以这么尝试：剪切steam安装目录的游戏文件到其他地方，关闭steam，试一下游戏能不能运行，能运行就没有drm

**Comment by shouhuanxiaoji** [comment]
https://rutracker.org/forum/viewtopic.php?t=5386953
这里可以下载到drm-free的gog版，建议你下载打包

**Comment by ghost** [comment]
@shouhuanxiaoji 执行. ./functions 有反应

**Comment by shouhuanxiaoji** [comment]
来群里交流
